services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_compose_specs.py

```diff
@@ -41,32 +37,58 @@ def _inject_proxy_network_configuration(
     target_container_spec["networks"] = container_networks
 
 
+EnvKeyEqValueList = List[str]
+EnvVarsMap = Dict[str, Optional[str]]
+
+
+class _environment_section:
```

Comment:
I think this brings no benefits. From my point of view it's harder to read and understand, it can bring issues when splitting on the equal sign (it should stop at the first occurrence).
Please revert it to how ti was.